core/chains/evm/config/chain_specific_config.go

```diff
@@ -164,6 +167,7 @@ func setChainSpecificConfigDefaultSets() {
 		ocrContractTransmitterTransmitTimeout: 10 * time.Second,
 		ocrDatabaseTimeout:                    10 * time.Second,
 		ocrObservationGracePeriod:             1 * time.Second,
+		ocr2AutomationGasLimit:                5300000, // 5.3M: 5M upkeep gas limit + 300K overhead
```

Comment:
Could use underscores for readability :shrug: (works in toml too)
```suggestion
		ocr2AutomationGasLimit:                5_300_000, // 5.3M: 5M upkeep gas limit + 300K overhead
```

Reply:
yup, much more readable